test/db_test_utils.hpp

```diff
@@ -423,53 +480,38 @@ class [[nodiscard]] tree_verifier final {
   [[nodiscard, gnu::pure]] constexpr Db &get_db() noexcept { return test_db; }
 
  private:
+  // Custom comparator is required for key_view.
+  struct comparator {
+    bool operator()(const key_type &lhs, const key_type &rhs) const {
+      if constexpr (std::is_same_v<key_type, unodb::key_view>) {
+        return unodb::detail::compare(lhs, rhs) < 0;
+      } else {
+        return lhs < rhs;
+      }
+    }
+  };
+
   Db test_db{};
 
-  std::unordered_map<unodb::key, unodb::value_view> values;
+  // Note: The hash map does not support key_view keys in the map.  So
+  // we need to switch over to the slower red/black tree for the
+  // ground truth map.
+  std::map<key_type, unodb::value_view, comparator> values;
+  // std::unordered_map<key_type, unodb::value_view> values;
```

Comment:
No commented out code

Reply:
removed.  but you might want to see if you can figure out how to keep using an unordered hash map with key_view.